server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -55,6 +55,11 @@ grant codeBase "${codebase.plugin-classloader}" {
 //// Everything else:
 
 grant {
+   permission java.util.PropertyPermission "*", "write";
```

Comment:
I'm fairly sure we don't want to grant ourselves permission to do this.

Reply:
Yes that was just a shortcut. But will need to allow write property permission for node_id & cluster_id if we decide to use System.setProperty